packages/openapi3/src/openapi.ts

```diff
@@ -473,6 +474,8 @@ function createOAPIEmitter(program: Program, options: ResolvedOpenAPI3EmitterOpt
     emitRequestBody(parameters, visibility);
     emitResponses(operation.responses);
 
+    isDeprecated(program, op) && (currentEndpoint.deprecated = true);
```

Comment:
Think using an `if` make this a bit more readable in this case

Reply:
sure, will do